shared/slices/slice_generic.go

```diff
@@ -0,0 +1,73 @@
+package slices
+
+// Type is the placeholder type that indicates a generic value.
+// when executed variables of this type will be replaced with
+// references to the specific types.
+// var GenericType generic.Type
+type Type interface{}
+
+// GenericItem of is of type empty interface in order
+// to hold any values.
+type GenericItem Type
+
+// GenericIntersection returns a new set with elements that exist in
+// both sets
```

Comment:
can you please add a period in the end for the function comments?

Reply:
yes sure. changing the comment

Reply:
Rectified